package.json

```diff
@@ -42,31 +42,30 @@
     "springroll": "^1.1.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.1",
-    "@babel/preset-env": "^7.0.0",
-    "@enlearn/eslint-config": "^1.2.0",
-    "babel-core": "^7.0.0-0",
-    "babel-jest": "^23.6.0",
-    "changelog-parser": "^2.5.0",
-    "eslint": "^5.6.0",
-    "eslint-config-prettier": "^3.0.1",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^21.22.0",
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "@enlearn/eslint-config": "^1.4.0",
+    "babel-jest": "^24.1.0",
+    "changelog-parser": "^2.6.0",
+    "eslint": "^5.13.0",
+    "eslint-config-prettier": "^4.0.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jest": "^22.2.2",
     "eslint-plugin-promise": "^4.0.1",
-    "husky": "^1.0.0-rc.14",
-    "jest": "^23.6.0",
-    "lint-staged": "^7.2.2",
-    "prettier": "^1.14.2",
-    "release-it": "^7.6.1",
-    "rimraf": "^2.6.2",
-    "rollup": "^0.66.0",
-    "rollup-plugin-babel": "^4.0.3",
-    "rollup-plugin-cleanup": "^3.0.0",
-    "rollup-plugin-commonjs": "^9.1.6",
-    "rollup-plugin-license": "^0.7.0",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-terser": "^3.0.0"
+    "husky": "^1.3.1",
+    "jest": "^24.1.0",
+    "lint-staged": "^8.1.3",
+    "prettier": "^1.16.4",
+    "release-it": "^10.1.0",
```

Comment:
If you haven't already, it would be worth double-checking everything in release-it 10. I know the deprecated the old way of pushing to a distribution repo, but that shouldn't affect this repo.

Reply:
Oh good catch. I will do some testing on it.

Reply:
After doing some testing and research, I've decided to revert release-it so as to not take on the burden of the breaking changes they've introduced. This is compounded by our desire to move to GH Actions or some other release mechanism that would make release-it redundant.